feat(`terraform_docs`): Add `terraform-docs` default markers support and describe how to migrate to them (#609)
antonbabenko/pre-commit-terraform: README.md

    * create a documentation file
    * extend existing documentation file by appending markers to the end of the file (see item 1 above)
    * use different filename for the documentation (default is `README.md`)

>     * use the same insertion markers as `terraform-docs`, i.e. `<!-- BEGIN_TF_DOCS -->...<!-- END_TF_DOCS -->`

Reviewer: ```suggestion
    * use the same insertion markers as `terraform-docs` by default.  Will be defaults in `v2.0`
      To migrate to `terraform-docs` insertion markers, run in repo root:
      
          ```bash
          grep -rl 'BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK' . | xargs sed -i 's/BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK/BEGIN_TF_DOCS/g'
          grep -rl 'END OF PRE-COMMIT-TERRAFORM DOCS HOOK' . | xargs sed -i 's/END OF PRE-COMMIT-TERRAFORM DOCS HOOK/END_TF_DOCS/g'
          ```
```

    ```yaml
    - id: terraform_docs
      args:
        - --hook-config=--path-to-file=README.md        # Valid UNIX path. I.e. ../TFDOC.md or docs/README.md etc.
        - --hook-config=--add-to-existing-file=true     # Boolean. true or false
        - --hook-config=--create-file-if-not-exist=true # Boolean. true or false

>         - --hook-config=--use-standard-markers=true     # Boolean. Defaults to false. Set to true for compatibility with terraform-docs

Reviewer: ```suggestion
        - --hook-config=--use-standard-markers=true     # Boolean. Defaults in v1.x to `false`. Set to true for compatibility with terraform-docs
```

    * create a documentation file
    * extend existing documentation file by appending markers to the end of the file (see item 1 above)
    * use different filename for the documentation (default is `README.md`)
    * use the same insertion markers as `terraform-docs` by default. It will be default in `v2.0`.  

>       To migrate to `terraform-docs` insertion markers, run in repo root:
>       
>       ```bash

Reviewer: ```suggestion
      To migrate to `terraform-docs` insertion markers, run in repo root:

      ```bash
```

>           ```bash
>           grep -rl 'BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK' . | xargs sed -i 's/BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK/BEGIN_TF_DOCS/g'
>           grep -rl 'END OF PRE-COMMIT-TERRAFORM DOCS HOOK' . | xargs sed -i 's/END OF PRE-COMMIT-TERRAFORM DOCS HOOK/END_TF_DOCS/g'
>           ```

Reviewer: ````suggestion
      ```bash
      grep -rl 'BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK' . | xargs sed -i 's/BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK/BEGIN_TF_DOCS/g'
      grep -rl 'END OF PRE-COMMIT-TERRAFORM DOCS HOOK' . | xargs sed -i 's/END OF PRE-COMMIT-TERRAFORM DOCS HOOK/END_TF_DOCS/g'
      ```
````
